tests/api.v2/model/Sample.test.js

```diff
@@ -0,0 +1,27 @@
+const { mockSqlClient } = require('../mocks/getMockSqlClient')();
+
+const Sample = require('../../../src/api.v2/model/Sample');
+
+const mockExperimentId = 'mockExperimentId';
+
+jest.mock('../../../src/sql/sqlClient', () => ({
+  get: jest.fn(() => mockSqlClient),
+}));
+
+
+describe('Sample apiv2 model', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it('Get samples works correctly', async () => {
```

Comment:
We could also mock responses/check the parameters passed to any helper functions called here (`replaceNullsWithObject`)